CMakeLists.txt

```diff
@@ -144,9 +144,23 @@ if(ASPECT_WITH_WORLD_BUILDER)
   CONFIGURE_FILE("${WORLD_BUILDER_SOURCE_DIR}/source/config.cc.in" "${CMAKE_BINARY_DIR}/world_builder_config.cc" @ONLY)
   LIST(INSERT TARGET_SRC 0 "${CMAKE_BINARY_DIR}/world_builder_config.cc")
 
+  # Move some file to the end for unity builds to make sure other file come
```

Comment:
Just as a clarification for me, this text is just to explain what happens, e.g. the code has not changed in functionality, or has it? As far as I can see you only moved the foreach code up, and accounted for that the source file contains the full name. Is that correct?

Reply:
Correct.